src/components/EquipmentExpansionList.tsx

```diff
@@ -31,20 +31,25 @@ const EquipmentExpansionList: FunctionComponent<EquipmentExpansionListProps> = (
   };
 
   // Expand panel when the currently selected category is this EquipmentExpansionList's cateogory (parentId)
-  const expanded = currentCategory === equipmentList[0].category ? true : false;
+  const expanded = currentCategory === (equipmentList[0].category.path || equipmentList[0].category.name)  ? true : false;
```

Comment:
The ternary is superfluous.  The comparison is already returning `true` or `false`.
```suggestion
  const expanded = currentCategory === (equipmentList[0].category.path || equipmentList[0].category.name);
```